.github/workflows/run-tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         python-version: [ 3.7, 3.8, 3.9 ]
-    timeout-minutes: 10
+    timeout-minutes: 12
```

Comment:
This is because we need to create many DHT instances (=> P2P instances) for averagers in tests. Another solution to consider is making the number of instances smaller.

Reply:
Do we still need 12min if we drop out test_overcrowded?

Reply:
We've reverted this change thanks to #334.